fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/ReferencesRdfContext.java

```diff
@@ -48,21 +57,36 @@ public ReferencesRdfContext(final FedoraResource resource,
         throws RepositoryException {
         super(resource, idTranslator);
         property2triple = new PropertyToTriple(resource.getNode().getSession(), idTranslator);
-        concat(putStrongReferencePropertiesIntoContext());
-        concat(putWeakReferencePropertiesIntoContext());
+        session = resource.getNode().getSession();
```

Comment:
The grandparent class of `ReferencesRdfContext` already has a `Session` data member:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel/src/main/java/org/fcrepo/kernel/utils/iterators/RdfStream.java#L52

It is probably not set in this scenario either. 
The comment above about passing in a Session at:
https://github.com/fcrepo4/fcrepo4/blob/master/fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/rdf/impl/NodeRdfContext.java#L52 
...could also resolve the issue of needing to produce a local session in this class.